.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,22 @@
+ ## What does it do?
```

Comment:
Codacy found an issue: [[no-heading-punctuation] Don’t add a trailing `?` to headings](https://app.codacy.com/gh/nezago/phone-history-store-frontend/pullRequest?prid=5869579)